testutil/accumulator.go

```diff
@@ -163,6 +167,15 @@ func (a *Accumulator) NFields() int {
 	return counter
 }
 
+// Wait waits for a metric to be added to the accumulator.
+// Accumulator must already be locked.
+func (a *Accumulator) Wait() {
```

Comment:
This was added so that tests don't have to call `sleep()` with some excessive value to avoid race conditions.